### PR TITLE
GOK-383 | Add audit log for successful redirects

### PR DESCRIPTION
**Pull request by mohan-13**
This PR adds Audit Service which adds audit logs for successful redirects to eSanjeevani portal.

### CODE REVIEW

File: src/main/java/org/bahmnihwc/esanjeevaniservice/service/OpenMRSAuditService.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OpenMRSAuditService {
+
+    @Autowired
+    private OpenMRSHttpClient openMRSHttpClient;
+
+    private static final String AUDIT_MESSAGE_FORMAT = "User %s accessed e-Sanjeevani for patient %s";
```
Comment (gsluthra) [inline src/main/java/org/bahmnihwc/esanjeevaniservice/service/OpenMRSAuditService.java:19]: Usually in Audit Log we don't put People's names, only Patient ID (to ensure PII data doesn't leak into Audit Log). Do you have a patient ID available to send? If not, then maybe its okay to send the patient name for now.

Comment (gsluthra) [reply src/main/java/org/bahmnihwc/esanjeevaniservice/service/OpenMRSAuditService.java:19]: Same is true for users.. we send username (and not user full name/etc).

Comment (mohan-13) [reply src/main/java/org/bahmnihwc/esanjeevaniservice/service/OpenMRSAuditService.java:19]: We are using only the login username here.

Comment (mohan-13) [reply src/main/java/org/bahmnihwc/esanjeevaniservice/service/OpenMRSAuditService.java:19]: For patient ID, we don't send ID as payload so using full name. And is stored in DB only, not shown in any reports

**Pull request merged**